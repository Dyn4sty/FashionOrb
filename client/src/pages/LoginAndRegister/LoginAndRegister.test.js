import React from "react";
import { shallow } from "enzyme";
import SignInAndRegister from "./LoginAndRegister";

it("should render SignInAndRegister component", () => {
  expect(shallow(<SignInAndRegister />)).toMatchSnapshot();
});
