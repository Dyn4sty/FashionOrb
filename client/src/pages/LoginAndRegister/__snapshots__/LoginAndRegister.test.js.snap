// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignInAndRegister component 1`] = `
<styled.div>
  <Connect(SignIn) />
  <Connect(SignUp) />
</styled.div>
`;
